packages/trip-form/src/util.tsx

```diff
@@ -269,9 +270,9 @@ function getTransitCombinedModeOptions(
             selectedCompanies.includes(modeCompanyUpper)),
         text: (
           <span>
-            <Suspense fallback={company || "Loading..."}>
+            <SafeSuspense fallback={company || "Loading..."}>
```

Comment:
Note that this is to accommodate tests only, and that no release will be triggered for this package.